src/actions/auth.js

```diff
@@ -105,7 +110,12 @@ export const loginWithFB = FBSDK => async (dispatch, getState) => {
       }
       break;
     default:
-      dispatchNotificationAndRollbarAndThrowError(dispatch, 'ER0006');
+      dispatchNotificationAndRollbarAndThrowError(
+        dispatch,
+        'ER0006',
+        null,
+        fbLoginResponse,
+      );
```

Comment:
跟這個PR本身無關，但我有想到這個method有更好的寫法
```suggestion
      dispatch(toastNotificationAndRollbarAndThrowError(
        'ER0006',
        null,
        fbLoginResponse,
      ));
```
```js
const toastNotificationAndRollbarAndThrowError = (
  errorCode,
  error,
  extra,
) => (dispatch, getState) => {
   // ...
};
```

Reply:
不行，這個方法不可行，我測試不 work
我理解這樣的話，`toastNotificationAndRollbarAndThrowError` 要直接綁 action creator